convert_lora_to_gguf.py

```diff
@@ -338,28 +338,39 @@ def generate_extra_tensors(self) -> Iterable[tuple[str, Tensor]]:
             def get_tensors(self) -> Iterator[tuple[str, Tensor]]:
```

Comment:
it seems like `lora_model_items_with_lora_tensors` and `lora_model_items_name` exist for no reason

You should change `350` and remove `344` and `345`:

```py
def get_tensors(self) -> Iterator[tuple[str, Tensor]]:
    tensor_map: dict[str, PartialLoraTensor] = {}

    for name, tensor in lora_model.items():
        if ("lora_" in name) or (".base_layer.weight" in name):
            if self.lazy:
                tensor = LazyTorchTensor.from_eager(tensor)
            base_name = get_base_tensor_name(name)
            is_lora_a = ".lora_A.weight" in name
            is_lora_b = ".lora_B.weight" in name
            if not is_lora_a and not is_lora_b:
                if ".base_layer.weight" in name:
                    continue

            if base_name in tensor_map:
                if is_lora_a:
                    tensor_map[base_name].A = tensor
                else:
                    tensor_map[base_name].B = tensor
            else:
                if is_lora_a:
                    tensor_map[base_name] = PartialLoraTensor(A=tensor)
                else:
                    tensor_map[base_name] = PartialLoraTensor(B=tensor)
        else:
            pass                

    for name, tensor in tensor_map.items():
        assert tensor.A is not None
        assert tensor.B is not None
        yield (name, cast(torch.Tensor, LoraTorchTensor(tensor.A, tensor.B)))
```

This is functionally the same and will be more performant.

please explain the reasoning behind the added complexity or if this is suitable.